confluent-kafka-connect-jdbc.spec.in

```diff
@@ -48,10 +48,13 @@ exit 0
 
 %files
 %defattr(-,root,root)
-/etc/kafka-connect-jdbc
+
+%doc
 /usr/share/java/kafka-connect-jdbc
 /usr/share/doc/kafka-connect-jdbc
 
+%config(noreplace) /etc/kafka-connect-jdbc
```

Comment:
According to the same doc, %config only accepts filename and not dirs. If this works, maybe the doc I am reading is outdated?

Reply:
According to another doc https://docs.fedoraproject.org/ro/Fedora_Draft_Documentation/0.1/html/RPM_Guide/ch09s05s03.html it does seem a dir could be used / guess newer version, but has wildcards at the end.

Guess best thing is to find the spec for our rpm distro, but I would image it would fall into either one of those... or not, since variety is a thing in Linux.